internal/app/go-filecoin/internal/submodule/syncer_submodule.go

```diff
@@ -16,7 +14,9 @@ import (
 	"github.com/filecoin-project/go-filecoin/internal/pkg/consensus"
 	"github.com/filecoin-project/go-filecoin/internal/pkg/net"
 	"github.com/filecoin-project/go-filecoin/internal/pkg/net/pubsub"
+	"github.com/filecoin-project/go-filecoin/internal/pkg/proofs/verification"
 	"github.com/filecoin-project/go-filecoin/internal/pkg/slashing"
+	fbig "github.com/filecoin-project/specs-actors/actors/abi/big"
```

Comment:
FWIW, unless you have an actual conflict, I don't think it's worth renaming this import.

Reply:
I have a medium strength opinion that it is good for developers to immediately know which package is being referenced because all the operations are similar and the naming collisions can be pretty confusing.

Maybe the use of math/big will virtually vanish as we move to this type and then I'll agree more with your point.